lib/user_location_plugin.dart

```diff
@@ -1,14 +1,33 @@
-
 import 'dart:async';
 
 import 'package:flutter/services.dart';
 
 class UserLocationPlugin {
+  static const _permissionEventChannel =
+      EventChannel("permission_event_channel");
```

Comment:
You can add this String into a String file, like the rest of string inside this class

Reply:
done